src/Temporalio.Extensions.OpenTelemetry/README.md

```diff
@@ -53,6 +53,38 @@ var client = await TemporalClient.ConnectAsync(new("localhost:7233")
 // Now client and worker calls are traced...
 ```
 
+If you are using dependency injection with `Temporalio.Extensions.Hosting`, you have to configure the client and
+worker separately. For example,
+
+```csharp
+// In Program.cs
+// NOTE: Remember to setup the tracer provider like in the above example.
+
+// Setup dependency injection
+var builder = WebApplication.CreateBuilder(args);
+
+// Choose one of the following options:
+//
+// Option 1: If you have a client and a worker, you can configure the client and
+// reuse the client in the worker.
+builder.Services
+    .AddTemporalClient(opts =>
+        {
+            opts.TargetHost = "localhost:7233";
+            opts.Namespace = "default";
+            opts.Interceptors = new[] { new TracingInterceptor() };
+        })
+    .AddHostedTemporalWorker("my-task-queue")
+// Option 2: If you have a worker but no client, you can configure the worker directly
```

Comment:
Sorry to be pedantic, but can you move this to a separate snippet? Can explain the options above the snippet if you'd like instead of comments, but as is, this snippet if taken as a whole is a bit broken.

Reply:
Done!